### PR TITLE
Update storage test suits for Windows

### PR DESCRIPTION
**Pull request by jingxu97**
Update storage test suits to enable some tests on Windows

Change-Id: I1f596bb9f7a0f41ae398aec43406ee5c753cdce3

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e/storage/testsuites/multivolume.go [hunk 1]
```diff
@@ -482,11 +482,11 @@ func TestConcurrentAccessToSingleVolume(f *framework.Framework, cs clientset.Int
 		ginkgo.By(fmt.Sprintf("Creating pod%d with a volume on %+v", index, node))
 		podConfig := e2epod.Config{
 			NS:            ns,
-			ImageID:       imageutils.DebianIptables,
 			PVCs:          []*v1.PersistentVolumeClaim{pvc},
-			SeLinuxLabel:  e2epv.SELinuxLabel,
+			SeLinuxLabel:  e2evolume.GetLinuxLabel(),
 			NodeSelection: node,
 			PVCsReadOnly:  readOnly,
+			ImageID:       e2evolume.GetTestImageId(imageutils.DebianIptables),
```
Comment (msau42) [inline test/e2e/storage/testsuites/multivolume.go:489]: why does this need to use iptables image?

File: test/e2e/storage/testsuites/volume_io.go [hunk 2]
```diff
@@ -141,7 +141,7 @@ func (t *volumeIOTestSuite) DefineTests(driver TestDriver, pattern testpatterns.
 		l.migrationCheck.validateMigrationVolumeOpCounts()
 	}
 
-	ginkgo.It("should write files of various sizes, verify size, validate content [Slow]", func() {
+	ginkgo.It("should write files of various sizes, verify size, validate content [Slow][LinuxOnly]", func() {
```
Comment (msau42) [inline test/e2e/storage/testsuites/volume_io.go:144]: Is it possible to make this testcase work on Windows? Should we open a tracking issue to track all the LinuxOnly test cases and see if it is possible to make it work for Windows?

File: test/e2e/storage/utils/utils.go [hunk 3]
```diff
@@ -69,14 +69,19 @@ const (
...
+		return f.ExecCommandInContainerWithFullOutput(pod.Name, pod.Spec.Containers[0].Name, "powershell", "/c", shExec)
+	}
+	return f.ExecCommandInContainerWithFullOutput(pod.Name, pod.Spec.Containers[0].Name, "/bin/sh", "-c", shExec)
+
 }
 
 // VerifyExecInPodSucceed verifies shell cmd in target pod succeed
 func VerifyExecInPodSucceed(f *framework.Framework, pod *v1.Pod, shExec string) {
 	stdout, stderr, err := PodExec(f, pod, shExec)
+	fmt.Printf("shExec %s", shExec)
```
Comment (msau42) [inline test/e2e/storage/utils/utils.go:82]: is this for debugging purposes?

File: test/e2e/storage/testsuites/multivolume.go (hunk 1 above)
Comment (jingxu97) [reply test/e2e/storage/testsuites/multivolume.go:489]: https://github.com/kubernetes/kubernetes/commit/f2bf2ab76ecfb43519e64dea2eef6051a5abaf31

Comment (jingxu97) [reply test/e2e/storage/testsuites/multivolume.go:489]: I think dd command has some special options that busybox cannot support

File: test/e2e/storage/testsuites/volume_io.go (hunk 2 above)
Comment (jingxu97) [reply test/e2e/storage/testsuites/volume_io.go:144]: It is possible. But need sometime to find a way to replace those linux commands. I will open an issue for it

**Comment by Jiawei0227** [comment]
Hmm, there seems to be a build issue?

File: test/e2e/storage/utils/utils.go (hunk 3 above)
Comment (jingxu97) [reply test/e2e/storage/utils/utils.go:82]: removed. thanks!

**Comment by jingxu97** [comment]
> Hmm, there seems to be a build issue?
Something wrong with import k8s.io/kubernetes/test/e2e/framework/pv". Not sure about the root cause, for now I will remove that dependency.

```
compilepkg: missing strict dependencies:
	/usr/local/google/home/jinxu/.cache/bazel/_bazel_jinxu/28e157fdac273189ba8ce989d815e18f/sandbox/linux-sandbox/3268/execroot/io_k8s_kubernetes/test/e2e/framework/volume/fixtures.go: import of "k8s.io/kubernetes/test/e2e/framework/pv"
No dependencies were provided.
```

**Comment by jingxu97** [comment]
finally, it passed tests. @Jiawei0227 @msau42 PTAL

**Comment by Jiawei0227** [comment]
I think it lgtm
/lgtm

**Pull request merged**